test_unstructured_ingest/test-ingest-slack.sh

```diff
@@ -8,21 +8,19 @@ OUTPUT_FOLDER_NAME=slack
 OUTPUT_DIR=$SCRIPT_DIR/structured-output/$OUTPUT_FOLDER_NAME
 DOWNLOAD_DIR=$SCRIPT_DIR/download/$OUTPUT_FOLDER_NAME
 
-if [ -z "$SLACK_TOKEN" ]; then
-   echo "Skipping Slack ingest test because the SLACK_TOKEN env var is not set."
-   exit 0
-fi
```

Comment:
we still need this, right?

Reply:
I got it to work without a token. Not sure if the one we're pulling from is public?

Reply:
Oh it's because the slack token is hard coded into the params that are passed in

Reply:
yep yep

Reply:
double checking we on the same page, the token for the test comes from env var `SLACK_TOKEN` which should be required to run this.

Reply:
when I try to run the test without the SLACK_TOKEN variable set I get: 
```
2023-07-28 13:33:57,697 SpawnPoolWorker-1 ERROR    Error: The request to the Slack API failed. (url: https://www.slack.com/api/conversations.history)
The server responded with: {'ok': False, 'error': 'not_authed'}```

Reply:
Looks like the latest script on `main` also has that requirement removed: [examples/ingest/slack/ingest.sh](https://github.com/Unstructured-IO/unstructured/blob/main/examples/ingest/slack/ingest.sh). Pretty sure whatever value was being pulled from `SLACK_TOKEN` now gets passed in explicitly via the `--token` parameter of the cli call: 
```python
PYTHONPATH=. ./unstructured/ingest/main.py \
        slack \
        --channels 12345678 \
        --token 12345678 \
        --download-dir slack-ingest-download \
        --structured-output-dir slack-ingest-output \
        --start-date 2023-04-01T01:00:00-08:00 \
        --end-date 2023-04-02
```